lib/puppet/provider/certmonger_certificate/certmonger_certificate.rb

```diff
@@ -1,4 +1,5 @@
 require 'ipaddr'
+require 'puppet/util/retryaction'
```

Comment:
The file is named `retry_action.rb`:
```suggestion
require 'puppet/util/retry_action'
```